WalletWasabi.Gui/ViewModels/StatusBarViewModel.cs

```diff
@@ -35,6 +37,8 @@ public class StatusBarViewModel : ViewModelBase
 		private WasabiSynchronizer Synchronizer { get; set; }
 		private SmartHeaderChain HashChain { get; set; }
 
+		private bool InLegalTrouble { get; set; } = false;
```

Comment:
`IsLegalTrouble` or `InLegalTrouble`?

Reply:
?

Reply:
I mean should that property be named `IsLegalTrouble` or is it correct as it is?

Reply:
It's a witty naming of a semaphore, which just wants to express if the legal related code part is being executed at the moment or not, so I think "IN legal trouble" as referring to "the execution is IN legal related code right now, so don't go into it again" is a more expressing name.

Reply:
Should not be in the viewModel.

Reply:
It's UI logic. `InLegalTrouble` is a sanity check semaphore on the UI thread, which is probably unnecessary, but better to have it than regret.